open-metadata-implementation/access-services/asset-manager/asset-manager-api/src/main/java/org/odpi/openmetadata/accessservices/assetmanager/properties/SchemaTypeProperties.java

```diff
@@ -111,6 +113,28 @@ public void setAuthor(String author)
     }
 
 
+    /**
+     * Return details of the expected usage of this technical element.
+     *
+     * @return string text
+     */
+    public String getUsage()
```

Comment:
I am curious about the thinking on why usage is in the technical metadata.

Reply:
Similar answer as to why displayName and description is in the technical metadata - it is possible that the tool is able to scavenge usage information from comments or similar descriptive information.

Reply:
This comment should really be addressed to the open metadata types - here it is in the documentation.  https://egeria-project.org/types/5/0501-Schema-Elements/

If it is not answering your questions you should make an update to the documentation when you find out the answers.